Content.Server/Chemistry/ReagentEffectConditions/JobCondition.cs

```diff
@@ -0,0 +1,48 @@
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Mobs;
+using Content.Shared.Mobs.Components;
+using Robust.Shared.Prototypes;
+using Content.Shared.Mind;
+using Content.Shared.Mind.Components;
+using Content.Shared.Roles;
+using Content.Shared.Roles.Jobs;
+using Content.Shared.Station;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.IoC;
+
+namespace Content.Server.Chemistry.ReagentEffectConditions
+{    
+    public sealed partial class JobCondition : ReagentEffectCondition
+    {
+        //TO DO: make this use protoid
+        [DataField] public List<string> Job = new List<string> {"Passenger"};
```

Comment:
remove the default value of passenger and just mark the field as required (`[DataField(required: true)]`)